src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -315,313 +347,326 @@ void func_80B67540(EnSth* this, PlayState* play) {
         case TEXT_STATE_5:
             if (Message_ShouldAdvance(play)) {
                 switch (play->msgCtx.currentTextId) {
-                    case 0x1134:
+                    case 0x1134: // (does not exist)
                         func_80151938(play, play->msgCtx.currentTextId + 1);
                         break;
 
-                    case 0x1132:
-                    case 0x113A:
-                    case 0x113F:
-                        func_80151938(play, 0x1133);
+                    case 0x1132:                     // Heard noise, came in to see
+                    case 0x113A:                     // Had no idea there was a basement here
+                    case 0x113F:                     // Heard noise... I never thought I'd find a place like this
+                        func_80151938(play, 0x1133); // did you find this place?
                         break;
 
-                    case 0x1133:
-                        func_80151938(play, 0x1136);
-                        func_80B670A4(this, 6);
+                    case 0x1133:                     // did you find this place?
+                        func_80151938(play, 0x1136); // I want to buy this place from you
+                        EnSth_ChangeAnim(this, STH_ANIM_PLEAD);
                         break;
 
-                    case 0x1136:
-                        SET_WEEKEVENTREG(WEEKEVENTREG_13_80);
+                    case 0x1136: // I want to buy this house from you
+                        SET_WEEKEVENTREG(WEEKEVENTREG_OCEANSIDE_SPIDER_HOUSE_COMPLETE);
```

Comment:
My intuition would think "complete" is when you've collected the 30th skull, but I could see it meaning you've collected the reward since this block is what sets the GI_ID. Maybe perhaps `WEEKEVENTREG_OCEANSIDE_SPIDER_HOUSE_COLLECTED_REWARD`? It might be nice to have a comment above that it's for the reward specifically at least.

I've taken a look and I think the other names are fine though. But if you want, would be nice to have a comment above as there's subtleties between the different flags (it's bit confusing), although not required